### ISSUE TITLE
BoxActivity.py:153 accessing member of None

### ISSUE THREAD

**Issue by jocassid** [w-a-r-m-inventory-system/Food-Pantry-Inventory#157]
Node this pair of lines.  The code is trying to access `self.activity.date_consumed` right after setting `self.activity` to `None`
```
                self.activity = None
                logger.debug(f'Act Box Fill: Activity box consumed: '
                      f'{self.activity.date_consumed}')
```

**Comment by jocassid** [comment]
There was a problem with the logging in this module (`logging.debug` and `logging.error` are not the preferred way to use the logging module).  I made this change in my `139_navigation_and_pallet_records` branch commit `2b68ce752e455db6ee3649107f525beb4164550f`

**Comment by deeppunster** [comment]
Fixed by #158
